reV/nrwal/nrwal.py

```diff
@@ -561,6 +563,11 @@ def _save_nrwal_misc(self, name, nrwal_config, output_mask):
             assert not any(value.variables), msg
             value = value.eval()
 
+        value = self._value_to_array(value, name)
+        self._out[name][output_mask] = value[output_mask]
+
+    def _value_to_array(self, value, name):
+        """Turn the input into numpy array if it isn't already."""
         if np.issubdtype(type(value), np.number):
             value *= np.ones(len(self._site_data))
```

Comment:
I realize that i wrote this but can you change it to:

value *= np.ones(len(self.analysis_gids), dtype=np.float32)

Reply:
Done and done!